src/cabin/cabin.utils.ts

```diff
@@ -0,0 +1,30 @@
+export function dateIsValid(dateString: string): boolean {
```

Comment:
You could use the yup library for validation if you like https://www.npmjs.com/package/yup

Reply:
https://github.com/dnt-frivillig/dnt-discord-bot/issues/12